packages/transformers/svg-react/src/SvgReactTransformer.js

```diff
@@ -35,6 +35,7 @@ export default (new Transformer({
 
     asset.type = 'jsx';
     asset.bundleBehavior = null;
+    asset.pipeline = undefined;
```

Comment:
By the way, I came up with one idea to fix #6748 on this PR.

As far as I know, from #4838, parcel-bundler does not transpile the module in node_modules because transpiling such modules will cause the unexpected error due to polyfill, and also not to transpile will lead to better performance. 

I think this policy is correct (for example, with webpack’s babel-loader, we usually specify `exclude: node_modules`). On the other hand, we may want to transpile some specific module in node_modules for legacy browser, so I propose `transpile` pipeline to specify to transpile its package even if this is in node_modules. With this pipeline, parcel can cover such use cases, and also we can fix #6748 simply by `Asset.pipeline = "transpile"`.

```javascript
import hoge from “transpile:module/should/transpiled/from/node_modules”;
```

I want feedback about this idea, and if this idea seems good I want to work on this 😄 .

Reply:
(or... should I propose it on RFC?)